source/common/memory/stats.cc

```diff
@@ -7,13 +7,13 @@
 namespace Memory {
 
 uint64_t Stats::totalCurrentlyAllocated() {
-  uint64_t value = 0;
+  size_t value = 0;
```

Comment:
yep, since the constraint is on `GetNumericProperty` I think is better to be explicit here for clarity if that makes sense; also this of course warns on Darwin since `__SIZE_TYPE__` is `unsigned long` and `uint64_t` is `unsigned long long`